Code/JavaWrappers/SubstanceGroup.i

```diff
@@ -1,8 +1,13 @@
+//
+// Created by Gareth Jones on 9/2/2020.
+//
+// Copyright 2020 Schrodinger, Inc
```

Comment:
Just to make it explicit, please add the usual bit of "this is part of the RDKit" boilerplate to this too:
```suggestion
// Copyright 2020 Schrodinger, Inc
//  @@ All Rights Reserved @@
//  This file is part of the RDKit.
//  The contents are covered by the terms of the BSD license
//  which is included in the file license.txt, found at the root
//  of the RDKit source tree.
```